src/main/java/duke/frontend/Parser.java

```diff
@@ -0,0 +1,57 @@
+package duke.frontend;
+
+import duke.task.TaskList;
+
+import java.util.Scanner;
+
+public class Parser {
+    private final TaskList taskList;
+    private static final String dashline = "--------------------------------------------------------------------------";
+    
+    public Parser(TaskList taskList) {
+        this.taskList = taskList;
+    }
+    
+    public void parseInputCommands() {
+        Scanner sc = new Scanner(System.in);
+        String next = sc.nextLine();
+        
+        while (!next.equals("bye")){
+        // "bye" breaks the while loop and causes the program to exit()
```

Comment:
Should the comment be indented relative to the next line rather than the previous? (See Java coding standard -> Comments)